airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -132,6 +132,8 @@ class KubernetesPodOperator(BaseOperator):  # pylint: disable=too-many-instance-
     :type do_xcom_push: bool
     :param init_containers: init container for the launched Pod
     :type init_containers: list[kubernetes.client.models.V1Container]
+    :param pod_template_file: path to pod template file
+    :type pod_template_file: str
     """
     template_fields = ('cmds', 'arguments', 'env_vars', 'config_file')
 
```

Comment:
What if the user pass image, name, namespace and yaml file? Which value will be used? I don't see the answer to this question in the documentation. I would prefer that the value from the YAML file be used, but this causes these arguments to be marked as optional.